ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -30,7 +30,6 @@ var DropdownModel = require('component/dropdown/dropdown');
 var MapContextMenuDropdown = require('component/dropdown/map-context-menu/dropdown.map-context-menu.view');
 var MapModel = require('./map.model');
 var MapInfoView = require('component/map-info/map-info.view');
-var MapSettingsDropdown = require('component/dropdown/map-settings/dropdown.map-settings.view');
```

Comment:
Should probably delete the dropdown/map-settings component because I don't think it is being used anywhere else